src/Main.ts

```diff
@@ -520,7 +522,7 @@ export class Main {
                 // Mark the room as active if we managed to join.
                 if (forRoom) {
                     forRoom.MatrixRoomActive = true;
-                    this.stateStorage?.trackRoom(ev.room_id);
+                    await this.stateStorage?.trackRoom(ev.room_id);
```

Comment:
This used to be a dangling promise that wasn't waited for.
I think it's meant to be waited for.

Reply:
Yeah, sounds good.